server/portal/settings/settings_default.py

```diff
@@ -259,6 +259,9 @@
     "hideSystemStatus": False,
     "hasUserGuide": True,
     "onboardingCompleteRedirect": "/workbench/",
+    "maxDescriptionLength": 800,
+    "maxTitleLength": 150,
+    "enableWorkspaceKeywords": False,
```

Comment:
I think we should default to `True` here and in the modals, and portals can opt out instead

Reply:
Got it!